gatsby-node.js

```diff
@@ -115,13 +117,130 @@ exports.sourceNodes = async ({ actions }) => {
   return
 }
 
-// exports.onCreateNode = ({ actions, node }) => {
-//   const { type } = node.internal
-//   if (type !== `Member`) return
-//   const { bio } = node
-//   const bioAsHtml = markdownProcessor.process(bio)
-//   node.bio = bioAsHtml
-// }
+exports.onCreateNode = ({ node, actions }) => {
+  const { type } = node.internal
+  if (type === 'GhostAuthor') {
+    if (node.slug !== 'data-schema-author') {
+      actions.createNodeField({
+        node,
+        name: `member___NODE`,
+        value: `Labs__Member__${node.slug}`,
+      })
+    }
+  }
+}
+
+const createTagPages = (tags, createPage) => {
+  const tagsTemplate = path.resolve(`./src/templates/tag.tsx`)
+
+  tags.forEach(({ node }) => {
+    const totalPosts = node.postCount !== null ? node.postCount : 0
+    const numberOfPages = Math.ceil(totalPosts / postsPerPage)
+
+    // This part here defines, that our tag pages will use
+    // a `/tag/:slug/` permalink.
+    node.url = `/tag/${node.slug}/`
+
+    Array.from({ length: numberOfPages }).forEach((_, i) => {
+      const currentPage = i + 1
+      const prevPageNumber = currentPage <= 1 ? null : currentPage - 1
+      const nextPageNumber =
+        currentPage + 1 > numberOfPages ? null : currentPage + 1
+      const previousPagePath = prevPageNumber
+        ? prevPageNumber === 1
+          ? node.url
+          : `${node.url}page/${prevPageNumber}/`
+        : null
+      const nextPagePath = nextPageNumber
+        ? `${node.url}page/${nextPageNumber}/`
+        : null
+
+      createPage({
+        path: i === 0 ? node.url : `${node.url}page/${i + 1}/`,
+        component: tagsTemplate,
+        context: {
+          // Data passed to context is available
+          // in page queries as GraphQL variables.
+          slug: node.slug,
+          limit: postsPerPage,
+          skip: i * postsPerPage,
+          numberOfPages: numberOfPages,
+          humanPageNumber: currentPage,
+          prevPageNumber: prevPageNumber,
+          nextPageNumber: nextPageNumber,
+          previousPagePath: previousPagePath,
+          nextPagePath: nextPagePath,
+        },
+      })
+    })
+  })
+}
+
+const createAuthorPages = authors => {
```

Comment:
Also do we need author pages or are member pages enough? What's the difference?

Reply:
you're right -- again left over copy/pasted cruft